From 7c5e43567b856da943067dd7ce8a4da302cb1998 Mon Sep 17 00:00:00 2001
From: Johnny Mnemonic <jm@machine-hall.org>
Date: Wed, 26 Mar 2025 12:46:45 +0100
Subject: [PATCH 67/94] ia64: handle "cpufreq: Remove cpufreq_generic_attrs"

See 486729c6012042c486db2a5e4d5dd034fb1d3f3c.

Also see for example 63c778aa1598c62971e7523c2a540da40cee1d70.
---
 drivers/cpufreq/ia64-acpi-cpufreq.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/cpufreq/ia64-acpi-cpufreq.c b/drivers/cpufreq/ia64-acpi-cpufreq.c
index f692729068f5..a996633ba093 100644
--- a/drivers/cpufreq/ia64-acpi-cpufreq.c
+++ b/drivers/cpufreq/ia64-acpi-cpufreq.c
@@ -326,7 +326,6 @@ static struct cpufreq_driver acpi_cpufreq_driver = {
 	.init		= acpi_cpufreq_cpu_init,
 	.exit		= acpi_cpufreq_cpu_exit,
 	.name		= "acpi-cpufreq",
-	.attr		= cpufreq_generic_attr,
 };
 
 
-- 
2.25.1

